monkestation/code/modules/virology/objects/vials.dm

```diff
@@ -0,0 +1,8 @@
+/obj/item/reagent_containers/cup/beaker/vial
+	name = "vial"
+	//icon = 'monkestation/code/modules/virology/icons/items.dmi'
```

Comment:
are these icon things meant to still be commented out?

Reply:
yea i dont have a vial sprite yet